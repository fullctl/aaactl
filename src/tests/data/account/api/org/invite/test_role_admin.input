{
  "client" : "api_client",
  "data": {
    "email": "invite@localhost",
    "is_admin_invite" : true
  }
}
