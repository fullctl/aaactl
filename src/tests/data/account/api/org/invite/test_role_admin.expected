{
  "status": 200,
  "response": {"data": [{"org_id": 2, "org_name": "test", "email": "invite@localhost", "id": 1, "status": "pending", "slug": "test", "user_name": "", "role" : "admin"}], "errors": {}, "profiling": {"queries": 0}}
}